### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by joephayes**
Partially resolves issue https://github.com/the-norman-sicily-project/interactive-map/issues/10 (builds and run tests)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - 8
+cache:
+  directories:
+    - node_modules
```
[the remark this answers is not in the document]
Comment (joephayes) [reply .travis.yml]: yup! Good catch. I switched to `yarn`

**Comment by RyanMarshall5765** [comment]
Looks good. Can squash and merge.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - 8
+cache:
+  yarn: true
+  directories:
+    - node_modules
+script:
+  - cp src/containers/ApiKey.js.sample src/containers/ApiKey.js
+  - yarn test
```
Comment (RyanMarshall5765) [inline .travis.yml]: Do we need yarn build here? Or will it auto run build?

**Comment by joephayes** [comment]
@RyanMarshall5765 - I added `yarn lint` to the script. I'll add the build once we start deploying.

File: .travis.yml (hunk 2 above)
Comment (joephayes) [reply .travis.yml]: No build step required here (no server dependency). Once we add deployment, we will call `yarn build` after the call to `yarn test`

**Pull request merged**